extras/kms/testing.go

```diff
@@ -0,0 +1,89 @@
+package kms
+
+import (
+	"context"
+	"database/sql"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/golang-migrate/migrate/v4"
+	"github.com/golang-migrate/migrate/v4/database"
+	"github.com/golang-migrate/migrate/v4/database/postgres"
+	"github.com/golang-migrate/migrate/v4/database/sqlite"
+	source "github.com/golang-migrate/migrate/v4/source"
+	"github.com/golang-migrate/migrate/v4/source/httpfs"
+	pgDriver "gorm.io/driver/postgres"
+
+	"github.com/hashicorp/go-dbw"
+	"github.com/hashicorp/go-kms-wrapping/extras/kms/v2/migrations"
+	"github.com/stretchr/testify/require"
```

Comment:
Nit: this import grouping appears inconsistent with other files.

Reply:
ty!

Reply:
not sure why the language server didn't fix that in vscode.